kernel-modules/build/Dockerfile

```diff
@@ -18,6 +20,14 @@ RUN apt-get update \
       pkg-config \
  ;
 
+RUN git clone https://github.com/google/googletest.git && \
```

Comment:
Seems like this should also be properly submodule'd.

Reply:
googletest is already a submodule of this repo. I am not sure what is the proper way to get it into this dockerfile.

Reply:
with `WORKDIR` and `COPY` directives

Reply:
Done